src/Core/TileMesh.js

```diff
@@ -11,10 +11,12 @@ import RendererConstant from '../Renderer/RendererConstant';
 import OGCWebServiceHelper, { SIZE_TEXTURE_TILE } from '../Provider/OGCWebServiceHelper';
 import { is4326 } from './Geographic/Coordinates';
 
-function TileMesh(geometry, params) {
+function TileMesh(layer, geometry, params) {
     // Constructor
     THREE.Mesh.call(this);
```

Comment:
the call to super could pass directly the geometry and material (with a material constructed at the call site)
`TileMesh(geometry, material, extent, level)`

Reply:
Done